app/js/questionNav.js

```diff
@@ -53,4 +53,22 @@ function prev() {
 }
 
 document.querySelector(".next").addEventListener("click", next)
-document.querySelector(".prev").addEventListener("click", prev)
\ No newline at end of file
+document.querySelector(".prev").addEventListener("click", prev)
+
+async function fillNav(HBTemplate) {
+    let template = Handlebars.compile(HBTemplate)
+    fetch("http://localhost:8080/question")
```

Comment:
It is possible to avoid an AJAX request - consider counting all the divs with class of 'question' in place of AJAX call.

Reply:
changed in 76ab4be